README.md

```diff
@@ -19,8 +19,8 @@ To get a local copy up and running follow these simple example steps.
 
 - Choose a directory
 - Open a Terminal
-- write: git clone https://github.com/storres20/TODOlistproject.git
-- get into to directory "TODOlistproject"
+- write: git clone https://github.com/storres20/todolist-test.git
+- get into to directory "todolist-test"
 - write: npm install

```

Comment:
- Please include steps on how to run your tests for your project 🙏 , this will help make your read me look more professional 👏